src/App.js

```diff
@@ -4,15 +4,16 @@ import './App.css';
 import { BrowserRouter as Router, Route, Link } from "react-router-dom";
 import "antd/dist/antd.css";
 import { Alumni } from "./alumni/alumni.component.js"
-
+import CustomForm from './alumni/form.js';
```

Comment:
Instead of custom form, perhaps call it something like AddAlumniForm

Reply:
changed name to AddAlumniForm